Documentation/network/servicemesh/ingress.rst

```diff
@@ -19,6 +19,25 @@ routing and for TLS termination.
     The ingress controller creates a service of LoadBalancer type, so
     your environment will need to support this.
 
+Cilium allows you to specify load balancer mode for the Ingress resource:
+
+- ``dedicated``: The Ingress controller will create a dedicated loadbalancer
+  for the ingress.
+- ``shared``: The Ingress controller will use a shared loadbalancer for all
+  Ingress resources.
```

Comment:
nit: Inconsistent capitalization and wording - perhaps we should pick `Ingress resource` and go with that for all cases?